AnnoyancesFilter/sections/annoyances.txt

```diff
@@ -1695,6 +1695,11 @@ thebell.io##.subs-only__overlay
 thebell.io#$#article.subs-only { height: auto !important; overflow: visible !important; }
 thebell.io##.follow > .follow__subtitle
 thebell.io##.follow > .follow-form
+thebell.io##app-subscription-sticky-dialog
+thebell.io##app-post-subscription
+thebell.io##app-widget-subscription
+thebell.io##.post_subscribe
```

Comment:
better make 
`thebell.io#$#.post_subscribe { display: none !important; }`
because some platforms do not support CSS rules and site will be a bit broken by the last rule.

Reply:
Done.

Reply:
Great, thanks.